book/eha.qmd

```diff
@@ -38,37 +40,50 @@ The *recurrent events setting* is as a special case of the multi-state setting b
-The survival outcome notation, $(T, \Delta)$ has to be extended to accomodate for the possible competing risks.
-Previously, $\Delta = 1$ if right-censoring is observed, this can be modified by multiplying $\Delta$ by the index of the event that occurs.
-Now, $\Delta \in \{0, 1, \ldots, k\}$ where $\Delta = 0$ still indicates censoring and $\Delta = j$ means the $j$th event occurred.
+In the competing risks setting, there is now an additional random variable $E\in \{1,\ldots,k\}$ with realisations $e$, which denotes one of $k$ competing events that can occur at event time $Y$.
+The survival outcome notation, $(T, \Delta)$ has to be extended to accomodate for the possibility of competing risks.
+Previously, we defined the status indicator $\Delta \in \{0,1\}$ for right-censored data.
+In the competing risks setting we have $\Delta = I(Y_i \leq C_I) \wedge E_i = e$, such that
+$\Delta \in \{0, 1, \ldots, k\}$ where $\Delta = 0$ indicates censoring as before and $\Delta = e$ means the $e$th event occurred.
 
-The predominate approach to competing-risks analysis is known as *cause-specific hazards*, where $h_{0e}$ is defined as the hazard for the transition from initial state $0$ to state $e =1,\ldots,k$:
+A common approach to competing-risks analysis is known as *cause-specific hazards*, where $h_{e}$ is defined as the hazard for the transition from initial state $0$ to state $e =1,\ldots,k$:
 
 $$
-h_{0e}(\tau) = \lim_{\Delta \tau \to 0} \frac{P(\tau \leq Y \leq \tau + \Delta \tau, E = e\ |\ Y \geq \tau)}{\Delta \tau}, \; e = 1, \dots, k.
+h_{e}(\tau) = \lim_{\Delta \tau \to 0} \frac{P(\tau \leq Y \leq \tau + \Delta \tau, E = e\ |\ Y \geq \tau)}{\Delta \tau}, \; e = 1, \dots, k.
 $$
 
-As competing events are mutually exclusive at any time $\tau$, it is useful to further define the:
+As competing events are mutually exclusive at any time $\tau$, it is useful to further define the *all-cause hazard* which is the hazard of any event occuring:
 
-- *all-cause hazard*, $h(\tau) = \sum_{e = 1}^{k}h_{0e}(\tau)$: which is the...
-- *all-cause cumulative hazard*: $H(\tau) = \int_{0}^\tau h(u)\ du = \sum_{e=1}^{k}\int_{0}^\tau h_{0e}(u)\ du = \sum_{e=1}^k H_{0e}(\tau)$, and
-- *all-cause survival probability*: $S(\tau) = P(Y \geq \tau) = \exp(-H(\tau))$
+$$
+h(\tau) = \sum_{e = 1}^{k}h_{e}(\tau)
+$$
+
+the *all-cause cumulative hazard*
+
+$$
+H(\tau) = \int_{0}^\tau h(u)\ du = \sum_{e=1}^{k}\int_{0}^\tau h_{e}(u)\ du = \sum_{e=1}^k H_{e}(\tau)
+$$
+
+and the *all-cause survival probability*
 
-Note that whilst the all-cause survival probability is defined in the same way as the standard survival probability, the interpretation is that *none* of the events occurred before $\tau$.
-The all-cause survival probability is also not directly estimated but instead calculated from the cause specific hazards.
+$$
+S(\tau) = P(Y > \tau) = \exp(-H(\tau))
```

Comment:
At this point I'm wondering if we can add a suffix to show we're in the competing risk setting, like S_CR, H_CR, etc.

Reply:
well, the cause-specific H_e has this, but H(\tau) here is the all cause hazard. S(\tau) is the all-cause surv prob. Would be unusual to add indices. Interpretation is also similar. Cumu hazard for any event, Prob to not experience any event.